EngineBay.DemoModule/Endpoints/TodoItemEndpoints.cs

```diff
@@ -0,0 +1,26 @@
+﻿namespace EngineBay.DemoModule.Endpoints
+{
+    public static class TodoItemEndpoints
+    {
+        private const string ItemBasePath = "/lists/{listId:guid}/items";
+
+        private static readonly string[] TodoItemTags = { ApiGroupNameConstants.TodoItem };
+
+        public static void MapEndpoints(RouteGroupBuilder endpoints)
+        {
+            endpoints.MapPost(
+                ItemBasePath,
+                async (CreateTodoItem handler, CreateTodoItemDto createTodoItemDto, Guid listId, CancellationToken cancellation) =>
+                {
+                    var command = new CreateTodoItemCommand(createTodoItemDto.Name, listId)
+                    {
+                        Description = createTodoItemDto.Description,
+                        DueDate = createTodoItemDto.DueDate,
+                    };
+                    var result = await handler.Handle(command, cancellation);
+                    return Results.Created($"{TodoListEndpoints.ListBasePath}/{listId}/items/{result.Id}", result);
```

Comment:
Doesn't this defeat the purpose of the basepath constant?

Reply:
Is there a way to substitute the ID into the base path, or should I make the item base path be "/item" and then add the listId parameter to the path in every mapping (or in another constant)?

Reply:
I tried adding it in another constant, not very pretty, but it exists

Reply:
Will we use listId in the other endpoints? Maybe list is specific to create and can be removed from the base path?

Reply:
Moved list id to the body after discussion